kovri.md

```diff
@@ -51,11 +44,9 @@ Network consensus ultimately determines who is allowed to relay traffic on the T
 Furthermore, correlation attacks are possible with VPNs, making it easy for large attackers to de-anonymize a user's traffic.
 
 So what can Alice, Bob, Charlie, and Dave do to mitigate these threats? They could use Kovri!
-If Alice uses Kovri to broadcast her transactions, no one will know the source of her transactions.
-Additionally, if they connect to the Monero network exclusively over Kovri, their peers will no longer see their IP addresses, making passive surveillance tactics impractical.
-Furthermore, the source of solved blocks are unknown when transmitted through Kovri, which will substantially improve Monero's censorship resistance.
+If they use Kovri to exclusively connect to the Monero network, no one will know their IP address, making passive surveillance impractical while substantially improving Monero's censorship resistance.
```

Comment:
This works, though the grammar would be a bit more clear if it said: "If they exclusively use Kovri to connect..."  And I would add a comma after "impractical."